content/survey_2022.ipynb

```diff
@@ -0,0 +1,4709 @@
+{
```

Comment:
<p>"popularity, with people from very different backgrounds now creating cool web apps."</p><p> </p><p>(want to be careful to avoid stereotyping)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/holoviz-dev/blog/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>